xrpl/models/transactions/amm_create.py

```diff
@@ -0,0 +1,78 @@
+"""Model for AMMCreate transaction type."""
+from __future__ import annotations
+
+from dataclasses import dataclass, field
+from typing import Dict, Optional
+
+from typing_extensions import Final
+
+from xrpl.models.amounts import Amount
+from xrpl.models.required import REQUIRED
+from xrpl.models.transactions.transaction import Transaction
+from xrpl.models.transactions.types import TransactionType
+from xrpl.models.utils import require_kwargs_on_init
+
+AMM_MAX_TRADING_FEE: Final[int] = 1000
```

Comment:
```suggestion
_AMM_MAX_TRADING_FEE: Final[int] = 1000
```

Reply:
It's not a private field since it's used both in `amm_create.py` and `amm_vote.py`